optimizer_wrapper.rb

```diff
@@ -20,6 +20,10 @@
 require_all 'lib'
 require_all 'util'
 
+RELATION_ZIP_CLUSTER_CAN_HANDLE = %i[minimum_day_lapse maximum_day_lapse minimum_duration_lapse maximum_duration_lapse
+                                     vehicle_trips vehicle_group_duration vehicle_group_duration_on_weeks
+                                     vehicle_group_duration_on_months vehicle_group_number].freeze
```

Comment:
![Screenshot from 2022-03-11 16-00-00](https://user-images.githubusercontent.com/20163961/157892780-505ce5dd-ad8c-4f60-a8da-17905b0dcb5c.png)

Alphabetical order and new lines for easy maintenance otherwise it leads to :point_up: 
```suggestion
RELATION_ZIP_CLUSTER_CAN_HANDLE = %i[
  maximum_day_lapse
  maximum_duration_lapse
  minimum_day_lapse
  minimum_duration_lapse
  vehicle_group_duration
  vehicle_group_duration_on_months
  vehicle_group_duration_on_weeks
  vehicle_group_number
  vehicle_trips
].freeze
```